### PR TITLE
[Community] handle auto refresh jwt token

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: additional_tests/supabase_backend_tests/test_user_elements.py [hunk 1]
```diff
@@ -162,3 +167,34 @@ async def test_sign_in_with_auth_token():
...
+    # use expired jwt token
+    expired_jwt_token = get_backend_client_expired_jwt_token(1)
+
+    # simulate auth using this token
+    session = mock.Mock(access_token=expired_jwt_token)
+    authenticated_client_1._listen_to_auth_events(
+       "SIGNED_IN", session
+    )
+
+    # now raising "APIError: JWT expired"
```
Comment (Herklos) [inline additional_tests/supabase_backend_tests/test_user_elements.py:190]: :+1:

File: octobot/cli.py [hunk 2]
```diff
@@ -229,6 +228,7 @@ async def _get_authenticated_community_if_possible(config, logger):
 
 
 async def _async_load_community_data(community_auth, config, logger, is_first_startup):
+    is_first_startup = True #tmp
```
Comment (Herklos) [inline octobot/cli.py:231]: Should we keep it ?

**Review by Herklos — COMMENTED** [review]
Nice fix & tests!

File: octobot/cli.py (hunk 2 above)
Comment (GuillaumeDSM) [reply octobot/cli.py:231]: :o no !

Comment (GuillaumeDSM) [reply octobot/cli.py]: up

**Review by Herklos — APPROVED** [review]

**Pull request merged**